debbuild

```diff
@@ -1945,18 +1955,23 @@ sub getreqs {
     # looked-for lib may not have the full soname version. (ie, it may
     # "just" point to one of the symlinks that get created somewhere.)
     next if grep { /\b$req/ } @intprovlist;
+
     # workaround for Ubuntu usrmerge, resolve symlinks
-    $req = "/usr$req" if $req =~ m|^/lib| && "/usr/lib" eq (readlink "/lib" || "");
+    # Firsly, we try to resolve dpkg path without modifications, then if it
+    # fails, we try to resolve it with /usr prefix (when /lib is a symlink to /usr/lib)
+    my $lib = dpkg_query_paths($req);
+    if(not(defined($lib)) && $req =~ m|^/lib| && "/usr/lib" eq (abs_path("/lib") || "")){
```

Comment:
I mean, do we make sure that it works with `/usr/lib/<triple>` but doesn't work with `/opt/<app>/lib/<triple>` or `/usr/lib/<app>/lib/<triple>`.

Reply:
I still do not sure if I understood you correctly. For now, the condition deals only with usrmerge, and we are adding `/usr` only in cases when `/lib` is a symbolic link to `/usr/lib` and `$req` starts with `/lib`.

`/opt/<app>/lib/<triple>` would not be translated to `/lib/opt/<app>/lib/<triple>` because it does not start with `/lib`

Reply:
I think we're good then.